KenticoCloud.Delivery/ContentItemsInRichText/ContentItemsInRichTextProcessor.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.ComTypes;
+using AngleSharp.Dom;
+using AngleSharp.Parser.Html;
+using AngleSharp.Parser.Xml;
+using AngleSharp.Services.Default;
+
+namespace KenticoCloud.Delivery.ContentItemsInRichText
+{
+    /// <summary>
+    /// Processor responsible for parsing richtext elements and resolving content items referenced in them using provided resolvers.
+    /// </summary>
+    public class ContentItemsInRichTextProcessor
```

Comment:
I don't like the *richtext* naming of the members... Can't we use something more generic and less verbose? E.g. `InlineContentItemsProcessor`?